AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/CampaignAdminControllerTests.cs

```diff
@@ -23,21 +23,40 @@ public class CampaignAdminControllerTests : InMemoryContextTest
         //delete this line when all unit tests using it have been completed
         private readonly Task taskFromResultZero = Task.FromResult(0);
 
-        [Fact(Skip = "NotImplemented")]
+        [Fact()]
```

Comment:
replace `[Fact()]` with `Fact[]` for the unit tests you implemented in this file